tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -1784,6 +1784,18 @@ private void handleIntervalFormats(XPathParts parts, String value, List<String>
             return;
         }
         examples.add(intervalFormat.format(FIRST_INTERVAL, later));
+        Set<String> relatedPatterns = RelatedDatePathValues.getRelatedPathValues(cldrFile, parts);
+        if (!relatedPatterns.isEmpty()) {
```

Comment:
I suggest checking relatedPatterns != null, or else adding @NotNull annotation to getRelatedPathValues. It looks like it currently can't return null, but better to make sure